style.css

```diff
@@ -9,6 +9,54 @@ body {
   background-color: #e5e5e5;
 }
 
+/* For drop down menu */
+.dropdown-menu {
+  position: absolute;
+  z-index: 10;
+  color: #fff;
+  background-color: #6070ff;
+  height: 110vh;
+  width: 100%;
+  mix-blend-mode: multiply;
+}
```

Comment:
Nice work implementing the overlay menu, however, as per the design it is suppose to stick to the screen till you click on the close button or any of the menu links. However, it dissappears from the screen when you scroll down as shown below. Kindly consider using `position: fixed` or something similar to correct this anomaly. 

### 1.

![Screenshot from 2023-01-31 08-52-14](https://user-images.githubusercontent.com/65233860/215714214-88e59796-67f5-4e37-8792-bd235d5e7895.png)

### 2. 

![Screenshot from 2023-01-31 08-52-23](https://user-images.githubusercontent.com/65233860/215714248-8fe5df24-9be2-4c13-b1b8-a203437972f6.png)

### 3.

![Screenshot from 2023-01-31 08-52-31](https://user-images.githubusercontent.com/65233860/215714272-0c4b1cbe-219d-4b76-a5d1-12fdb27b7c9c.png)

### 4.

![Screenshot from 2023-01-31 08-52-40](https://user-images.githubusercontent.com/65233860/215714293-eb51d92d-d4bd-4b1e-8cf7-572724f1d4a7.png)